korgw/src/commonMain/kotlin/com/soywiz/korag/shader/gl/GlslGenerator.kt

```diff
@@ -115,6 +115,7 @@ class GlslGenerator constructor(
 
         //println("types.funcRefs=${types.funcRefs}")
         val customFuncs = funcs.filter { it.ref.name in types.funcRefs }.reversed().distinctBy { it.ref.name }
+        //TODO: is it relevant to visit mainFunc for each existing function ?
         for (func in funcs) types.visit(mainFunc)
```

Comment:
@soywiz found this on the gl implementation, not sure to understand the point.

Reply:
This is probably a typo